readme.md

```diff
@@ -99,7 +99,7 @@ GitHub Enterprise is also supported. More info in the options.
 - Copy the canonical link of a file with [the <kbd>y</kbd> hotkey](https://help.github.com/articles/getting-permanent-links-to-files/).
 - Indent text with <kbd>tab</kbd> inside comment fields (<kbd>shift</kbd> <kbd>tab</kbd> for original behavior).
 - [Use the pull request title as the commit title when merging with `Squash and merge`](https://github.com/sindresorhus/refined-github/issues/276).
-- [Directly view linked gists inside comments.](https://user-images.githubusercontent.com/6978877/33911900-c62ee968-df8b-11e7-8685-506ffafc60b4.PNG)
+- [Directly view linked gists inside comments.](https://user-images.githubusercontent.com/6978877/33911900-c62ee968-df8b-11e7-8685-506ffafc60b4.png)
```

Comment:
This new URL doesn't work. If you want to change case you'll have to re-upload it

Reply:
Woah, that's weird. Did GitHub generate `.PNG` URLs in the past? I'll just remove the commit.

Reply:
No, GitHub just preserves the extension as-is, so if you upload `FILE.PNG` you'll get `2cb44.PNG`